Libraries/LibC/pwd.cpp

```diff
@@ -125,4 +125,19 @@ struct passwd* getpwent()
     strncpy(__pwdb_entry->shell_buffer, e_shell.characters(), PWDB_STR_MAX_LEN);
     return __pwdb_entry;
 }
+
+int putpwent(const struct passwd* p, FILE* stream)
+{
+    // glibc does some more advanced checks on p's members. We should too eventually.
+    // For now, checking for NULL and setting EINVAL should be good enough :)
+    if(p == NULL || stream == NULL || p->pw_name == NULL || p->pw_dir == NULL || p->pw_gecos == NULL || p->pw_shell == NULL)
```

Comment:
Missing space after `if`. Please use `clang-format`.
Also, please use `nullptr` instead of `NULL`, or even better, check `!p || !stream || ...` :)

Furthermore, we should at the very least check that we're not putting `:` or `\n` characters in fields, since that will bork parsing.

Reply:
> Also, please use nullptr instead of NULL

I was going to sorry, but my reasoning was it was in `LibC`, so, yknow..